CMakeLists.txt

```diff
@@ -161,6 +161,7 @@ add_subdirectory(core)
 add_subdirectory(engine)
 add_subdirectory(api)
 add_subdirectory(tools)
+add_subdirectory(scripting)
```

Comment:
Imo the dir should be named bindings - we may want to bind 'full' langs like rust later on